cpp/density/RDF.cc

```diff
@@ -22,6 +22,10 @@ RDF::RDF(unsigned int bins, float r_max, float r_min, bool normalize)
     {
         throw std::invalid_argument("RDF requires r_max to be positive.");
     }
+    if (r_min < 0)
+    {
+        throw std::invalid_argument("RDF requires r_min to be zero or positive.");
```

Comment:
We use the term "non-negative" in other parts of the code. Let's keep consistent with that naming convention. Examples below.
```suggestion
        throw std::invalid_argument("RDF requires r_min to be non-negative.");
```

https://github.com/glotzerlab/freud/blob/eb2a97bac302ed45236f26e0a7e6a0346071c579/cpp/locality/PeriodicBuffer.cc#L20

https://github.com/glotzerlab/freud/blob/eb2a97bac302ed45236f26e0a7e6a0346071c579/cpp/order/SolidLiquid.cc#L17-L19

Reply:
okay